### DISCUSSION TITLE
Question about tempora notation files

### DISCUSSION THREAD

**Discussion by d4h0y0sr** [DivinumOfficium/divinum-officium#3855] [Developers corner]
Hi, I see that according to documentation in the page, "r" notation is for Sancti, and means "1960", nevertheless I have an issue because I select in the page following settings: "Reduced 1955", day "6-09-2024", language spanish, and its not reading the file "Pen03-0.txt", but "Pen03r.txt" inside Tempora folder. So the notation also applies to tempora? it still has the same meaning?

**Comment by FAJ-Munich** [comment]
You can find the file names used in Tempora for 1955 and 1960 in `web/www/Tabulae/Tempora/1960.txt`
It's mainly reflecting the abolishment of the octaves and the changes in Holy Week.
